### PR TITLE
Try moving terminal page contents into the running tab.

### PR DESCRIPTION
**Pull request by jdfreder**
Lets us see what @ellisonbg was talking about in the dev meeting today.

### CODE REVIEW

**Comment by jdfreder** [comment]
Now that  #7242 was merged, I went ahead and used a less variable for the header bg color @Carreau .

File: IPython/html/static/tree/less/tree.less [hunk 1]
```diff
@@ -179,3 +179,43 @@ ul#new-menu {
...
+                padding: 0px;
+
+                .list_container {
+                    margin-top: 0px;
+                    margin-bottom: 0px;
+                    border: 0px;
+                    border-radius: 0px;
+
+                    .list_item {
+                        border-bottom: 1px solid #ddd;
```
Comment (ellisonbg) [inline IPython/html/static/tree/less/tree.less]: Do we have a less variable for the `#ddd` color?

**Comment by ellisonbg** [comment]
On the Files and Cluster tab there is `8px` below the refresh button and the table below. On the Running tab there is much more space. There are a couple of ways you could do this in the css, but I think the Running Tab should have `8px` between the Refresh and first panel and then an "appropriate" amount of spacing between the two panels.

**Comment by ellisonbg** [comment]
Otherwise this looks great!

**Comment by ellisonbg** [comment]
Fixes #7234

**Comment by fperez** [comment]
This looks great, but I wonder: should terminals go below notebooks instead of above?  Notebooks are likely to be far more common...

The counter argument is, obviously, b/c the nb list is likely to be longer, it may be handy to have the (likely few) terminals available first without having to scroll down a long list of notebooks.

Decisions, decisions...

**Comment by ellisonbg** [comment]
@jdfreder did you see the review comments I left?

File: IPython/html/static/tree/less/tree.less (hunk 1 above)
Comment (jdfreder) [inline IPython/html/static/tree/less/tree.less]: Yes, there's a bootstrap variable for that, should be fixed.

**Comment by jdfreder** [comment]
> On the Files and Cluster tab there is 8px below the refresh button and the table below. On the Running tab there is much more space. There are a couple of ways you could do this in the css, but I think the Running Tab should have 8px between the Refresh and first panel and then an "appropriate" amount of spacing between the two panels.

Should be fixed in last commit.

> @jdfreder did you see the review comments I left?

Yup, just now.  Sorry I've been AFK for a little over a week now.

**Comment by jdfreder** [comment]
Rebased

**Comment by ellisonbg** [comment]
```
.list_toolbar .tree-buttons {
padding-top: 1px;
}
```

**Comment by ellisonbg** [comment]
And check the other weird padding thing...

**Comment by ellisonbg** [comment]
Looks like the double border line on the terminals page is back...maybe bad rebase?

**Comment by ellisonbg** [comment]
Dashboard colors look good.

**Comment by jdfreder** [comment]
@ellisonbg I addressed your comments in my last commit!

**Comment by ellisonbg** [comment]
This looks good, merging.

**Pull request merged**